templates/nginx/vhost.conf

```diff
@@ -26,7 +26,7 @@ server {
 {% endif %}
 
   location / {
-    root   /var/www/bigbluebutton-default;
+    root   {{ bbb_nginx_root | regex_replace('\\/$', '' }};
```

Comment:
(same) `regex_replace` function is missing the closing parenthesis `)`

Reply:
Fixed in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/bef7c63da4f638dc42669b3fa78460c6bf64fc35